### PR TITLE
Feature/project settings

### PR DESCRIPTION
**Pull request by TMCamp**
Jira: https://jiraent.cms.gov/browse/ISPGBSS-673

I updated the base template to have a more consistent header inline with the designs, moved the project_settings.html file into the /templates/project directory (and replaced the _ with a - so that it is more consistent with the other template files).

### CODE REVIEW

File: templates/project/project-settings.html [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+  </li>
+  <li class="usa-breadcrumb__list-item usa-current" aria-current="Project Settings">
+    <span>Project Settings</span>
+  </li>
+{% endblock %}
+
+{% block body_content %}
+
+  <section class="margin-bottom-4">
+    <p class="usa-intro">
```
Comment (amyd11) [inline templates/project/project-settings.html:31]: I know we talked about which uswds style to use for this text, but it seems like the usa-intro has a set font that deviates from the designs. I think we might need to custom size it instead.

**Review by amyd11 — APPROVED** [review]
So many good style updates- It's going to be so much easier to maintain with the updated page title and margins.

We might want to also delete the old `project_settings.html` file with this PR

File: templates/project/project-settings.html (hunk 1 above)
Comment (TMCamp) [reply templates/project/project-settings.html:31]: Ah, good catch!

Comment (TMCamp) [reply templates/project/project-settings.html:31]: I added a font-family override to the cms.scss file. It is already in branding.

**Pull request merged**